### DISCUSSION TITLE
Cables - as units with numbering & pathing

### DISCUSSION THREAD

**Discussion by Kyetech-IT-Manager** [opendcim/openDCIM#1385] [Ideas]
Could you add cable as units to add to the system so cable numbering can be part of the path from node to node, with all the patch panels, switches, cables showing up on the path connection display

**Comment by lmwayne** [comment]
I am looking for a DCIM solution for my workplace - we support 3 small data centers. We need cable numbers for all the cables at the site (currently managed with Excel spreadsheets). Does openDCIM support cable numbers?

**Reply by alex001x** [reply]
OPENDCIM in the table of cable device.php connections, the "note" column allows to identify the cable number this reference can be used in the cable search in views path connection (paths.php) Search by path identifier

Alex,
![image](https://github.com/user-attachments/assets/6670f210-4a44-4e24-b808-52ef6acbf654)

![image](https://github.com/user-attachments/assets/041072ae-4883-42f6-adb8-3030fc58eac7)
